### DISCUSSION TITLE
🔗 Calculated Fields in Elementor and other features in elementor forms

### DISCUSSION THREAD

**Discussion by ayoubkhan558** [elementor/elementor#22169] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Currently, Elementor does not offer a way to calculate fields within the form builder. This limits the functionality of forms and requires users to export form data to calculate values in external software. This process can be time-consuming and error-prone.
![image](https://github.com/user-attachments/assets/df5153e6-8fef-49ce-b67c-0cda4471919f)
![image](https://github.com/user-attachments/assets/a880fabb-01e6-4f0e-98ef-dda14875c7ed)

### Describe the solution you'd like

We would like to request the addition of calculated fields in the Elementor form builder. This would allow users to perform calculations and manipulate form data within the form builder itself, saving time and reducing the risk of errors.

Calculated fields could be implemented in several ways, such as by allowing users to add mathematical formulas directly into form fields or by providing a separate calculated field type that users can add to their forms. The calculated field could also include support for dynamic values, allowing users to reference other form fields or custom variables in their calculations.

Overall, the addition of calculated fields would greatly enhance the functionality of Elementor forms and improve the user experience for many users. We believe that this feature would be a valuable addition to the Elementor plugin and would greatly benefit the Elementor community.

### Describe alternatives you've considered

[Forminator](https://g-ladybird-rupu.instawp.xyz/wp-admin/plugin-install.php?tab=plugin-information&plugin=forminator&TB_iframe=true&width=600&height=550)

**Comment by ghost** [comment]
+1

**Comment by nickdgreen** [comment]
+1

**Comment by mikeecho2** [comment]
This has been a request from the Elementor PRO Users for several years now, but sadly it has become very obvious that Elementor is too interested in becoming a Hosting company now, and only add features to Elementor that NO ONE ASKED FOR, like their new cash cow AI system.

So I would not hold my breath that this VERY BASIC FORM FEATURE like CALCULATED FIELDS & DYNAMIC FIELD DISPLAY CONDITIONS will ever be added to Elementor forms

It has been more than 3 YEARS, and I believe it was even asked for by the community for even longer ago
https://github.com/elementor/elementor/issues/10386

**Comment by UMSOORG** [comment]
Plus one for calculated forms

**Comment by catmur-neame** [comment]
yes please

**Comment by sbu-scnl** [comment]
Yes please
